Week1/1.txt

```diff
@@ -0,0 +1,18 @@
+1)Write a python script to enter any number and check its prime or not.
+
+n = 9
+if(n > 1):
+	for k in range(2, int(sqrt(n)) + 1):

```

Comment:
why you use "sqrt" ?

Reply:
It has a major use in the formula for roots of a quadratic equation.